guess-the-country/country-backend/country/main.py

```diff
@@ -19,6 +19,6 @@
 http_client.set_up(app)
```

Comment:
```suggestion
tracking.instrument(app)
```

Das sollte hier oben passieren, der `http_client` wird dabei gleich mit initialisiert.